README.gcp.md

```diff
@@ -55,8 +22,9 @@ make run_local_tests
 
 
 ```
-pip3 install -r requirements.txt
 cd ~/bridge-hackathon/DDS/
+pip3 install -r requirements.txt
+make libdds-build
```

Comment:
Looks good. Even better, we can add a make target to run the local server, per my other comment.